testing/runner.ts

```diff
@@ -96,7 +97,16 @@ export class Runner {
               console.error(`\n    Actual:\n`);
               console.error(indent(JSON.stringify(result.err.actual, null, 4), 8));
             }
-            console.error("\n");
+            console.error(`\n    Overall diff:`);
```

Comment:
In some cases, either expected or actual is missing, can you wrap this in an if statement?

Reply:
Ah good catch